microsoft-edge/webview2/how-to/winrt-from-js.md

```diff
@@ -305,6 +305,16 @@ Next, pass the WinRT object from the native side of the host app to the web side
 
    This method calls `AddHostObjectToScript`.
 
+   In the line AddHostObjectToScript(“Windows”, Windows is the top level namespace. So you can add your own third party one as well like the following example    :
```

Comment:
```suggestion
   In the line `AddHostObjectToScript(“Windows”, ...`, Windows is the top level namespace. So you can add your own third party one as well like the following example    :
```

Reply:
I tweaked the @david-risney  Suggestion above.  We can probably close the @Syul968 Suggestion.